docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,159 @@
+installer, accepting the defaults. In your terminal, run:
+``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
+
+On Windows, the default is the *.msi* installer. Download and execute the
+installer. During the install process, look for an option labeled
+**Set JAVA_HOME variable**. Ensure it is enabled; if a red letter *X* precedes it,
+click the red letter *X* and choose **Will be installed on local hard drive**.
+
+On Linux, the default is a _.tar.gz_ file. Download it and unpack it to a
+directory of your choosing. Set the *JAVA_HOME* environment variable to the
+directory containing *bin*; for example, if you unpacked the *.tar.gz* file
+into ``~/java/1.8/``, at the time of writing, you would need to run
+``export JAVA_HOME=$HOME/java/1.8/jdk8u242-b08``.
+
+Testing your Java 8 environment
+-------------------------------
+
+To make sure that the Java compiler is available via the *JAVA_HOME*
+environment variable, and that it is version 8, run the following command.
+
+We test *javac* here, not *java*, because if *javac* works, so will *java*.
+
+If you get a "Command not found" error on macOS or Linux, look for the
+``export`` command in the previous section, and run it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/bin/javac" -version
+      javac 1.8.0_242
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/javac" -version
+      javac 1.8.0_242
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>"%JAVA_HOME%\bin\javac" -version
+      javac 1.8.0_242
+
+Create an Android app and compile it
+====================================
+
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. First, you run the ``create``
+command. This downloads an Android app template and adds your Python code to it.
+
+.. code-block:: bash
+
+  (beeware-venv) $ briefcase create android
+
+  [helloworld] Generating application template...
+  Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+  ...
+  [helloworld] Installing support package...
+  ...
+  [helloworld] Installing dependencies...
+  ...
+  [helloworld] Installing application code...
+  ...
+  [helloworld] Installing application resources...
+  ...
+  [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project configured to use gradle, your application code, and a support
+package containing the Python interpreter and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take more than two minutes the first time, depending on
```

Comment:
... In the Bay area maybe. It took closer to 10 minutes for me. :-)

It would also be desirable to show what "working, but slowly" looks like - because the Gradle UI around this is *shockingly* bad.

Reply:
For now, I'll do a very lightweight version of this: I'll state it in text in the paragraph. Does that do the job? I'm fearful of making this doc super long, and I prefer to be lazy and not configure gradle into the error case then get the exact output. I can also be non-lazy and get the exact output if you prefer.

Reply:
We're discussing more below, so I'll resolve this one.